Makefile

```diff
@@ -318,9 +318,10 @@ swagger: config/swagger.yaml $(SWAGGER_CODEGEN_CLI) $(SWAGGER_ENDPOINTS_SPEC)
 	@( mkdir -p $(HTTP_APP)/priv && cp $(SWTEMP)/priv/swagger.json $(HTTP_APP)/priv/; )
 	@( cd $(HTTP_APP) && $(MAKE) updateswagger; )
 	@rm -fr $(SWTEMP)
-	@$(SWAGGER_CODEGEN) generate -i $< -l python -o $(SWTEMP)
+	@$(SWAGGER_CODEGEN) generate -i $< -l python -o $(SWTEMP) --import-mappings GAObject="from hack_ga_object import GAObject"
```

Comment:
```suggestion
	@$(SWAGGER_CODEGEN) generate -i $< -l python -o $(SWTEMP) --import-mappings GAObject="from swagger_client.models.hack_ga_object import GAObject"
```

Reply:
@ThomasArts As you noticed, using Python3 enables reproducing on other archs than Win. With this patch it passes locally for me (macOS).